PowerEditor/src/Parameters.cpp

```diff
@@ -2148,55 +2148,64 @@ void NppParameters::updateLangXml(TiXmlElement* mainElemUser, TiXmlElement* main
 					int nWordsAdded = 0;
 					TiXmlNode* pChildModel = keywordsFromModel->FirstChild();
 					std::wstring wsTextModel = pChildModel ? pChildModel->Value() : L"";
-					if (!wsTextModel.empty())
+					if (!pChild && pChildModel)

```

Comment:
2/2
In case of pChildModel is NULL, the crash may still happen.
I would suggest the following code:
```cpp
if (!pChild)
{
    if (pChildModel)
    {
        TiXmlNode* p_clone = pChildModel->Clone();
        mapUserKeywords[modelKeywordsName]->LinkEndChild(p_clone);
    }
}
else
{
    ...
```